db/db.go

```diff
@@ -0,0 +1,71 @@
+package db
+
+import (
+	"gopkg.in/mgo.v2"
+	"log"
+	"github.com/AITestingOrg/calculation-service/models"
+	"net/http"
+	"encoding/json"
+)
+
+type TripDao struct {
+	Server string
+	Database string
+}
+var MgoSession *mgo.Session
+
+func ErrorWithJSON(w http.ResponseWriter, message string, code int) {
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	w.WriteHeader(code)
+	log.Printf( "{message: %q\nheader: }", message, w)
+}
+
+// Establish connection with db
+func init() {
+	session, err := mgo.Dial("mongo:27017")
+
+	if err != nil {
+		log.Fatal("cannot dial mongo", err)
+	}
+	session.SetMode(mgo.Monotonic, true)
+	MgoSession = session
+}
+
+// Find a trip by it id
+//func (m *TripDao) FindById(id string) (models.Cost, error) {
+//	var cost Cost
+//	err := MgoSession.C(COLLECTION).FindId(bson.ObjectIdHex(id)).One(&cost)
+//	return trip, err
+//}
```

Comment:
If this code is unnecessary please remove the comments

Reply:
code was removed